docs/ref/settings.txt

```diff
@@ -385,6 +385,11 @@ where ``reason`` is a short message (intended for developers or logging, not for
 end users) indicating the reason the request was rejected.  See
 :doc:`/ref/csrf`.
 
+The default implementation ``django.views.csrf.csrf_failure`` takes an additional
+keyword parameter ``template_name`` that defaults to ``'csrf_fail.html'``. If a
```

Comment:
This template name doesn't match the one actually used in the code (``csrf_failure.html``). Personally I prefer the shorter version, but don't really care.